cmd/kubelet/app/options/options.go

```diff
@@ -36,8 +37,20 @@ import (
 const (
 	defaultRootDir             = "/var/lib/kubelet"
 	experimentalFlannelOverlay = false
+
+	defaultPodInfraContainerImageName    = "gcr.io/google_containers/pause"
+	defaultPodInfraContainerImageVersion = "2.0"
```

Comment:
As I understand it, it's still possible to change the value by passing `--pod-infra-container-image` to `kubelet`. This is just the default.

Or am I missing something?